lib/standalone.js

```diff
@@ -175,11 +175,14 @@ module.exports = function (options) {
 		const cwd = _.get(globbyOptions, 'cwd', process.cwd());
 		const absolutePath = !path.isAbsolute(entry) ? path.join(cwd, entry) : path.normalize(entry);
 
+		entry = entry.replace(/\\(?![[\]()])/g, '/');
```

Comment:
Perhaps you can add a unit test on this behaviour, to make sure it doesn't break in the future

Reply:
Added unit test for windows style backslash in paths